docs/content/guides/migrations/from-step-launchers-to-pipes.mdx

```diff
@@ -0,0 +1,333 @@
+---
+title: "Migrating From Step Launchers to Dagster Pipes | Dagster Docs"
+description: "Learn how to migrate from step launchers to Dagster Pipes in Dagster."
+---
+
+# Migrating From Step Launchers to Dagster Pipes
```

Comment:
This guide is specifically demonstrating how to migrate Spark workloads.

Might there be a more general example for a migration guide, and then we can use this as a separate migrating Spark workloads guide?

Reply:
We don't have any Step Launchers other than Spark so that's basically a synonym